src/defaults.mligo

```diff
@@ -4,11 +4,11 @@
 #include "types.mligo"
 #include "proposal.mligo"
 
-let validate_proposal_flush_expired_time (data : initial_config_data) : unit =
-  if data.proposal_expired_time <= data.proposal_flush_time then
-    failwith("'proposal_expired_time' needs to be bigger than 'proposal_flush_time'.")
-  else if data.proposal_flush_time <= (data.period.length * 2n) then
-    failwith("'proposal_flush_time' needs to be more than twice the 'period' length.")
+let validate_proposal_flush_expired_level (data : initial_config_data) : unit =
+  if data.proposal_expired_level.blocks <= data.proposal_flush_level.blocks then
+    failwith("proposal_expired_level needs to be bigger than proposal_flush_level")
+  else if data.proposal_flush_level.blocks <= (data.period.blocks * 2n) then
+    failwith("proposal_flush_level needs to be more than twice the voting_period length")
```

Comment:
(see previous comment) ... here I suggest:
```suggestion
  else if data.proposal_flush_level.blocks < (data.period.blocks * 2n) then
    failwith("proposal_flush_level needs to be more than twice the 'period' length.")
```

Reply:
Changed.

Reply:
@sras can you also please check that there's no ambiguity in the documentation?

Reply:
@sras can you also please check that there's no ambiguity in the documentation?

Reply:
Checked, and it seems that the current description can have some ambiguity, because it says something like "x levels after which a propsal is raised". Because it does not make it clear if the block at which the proposal is raised is counted as one.

So I have now introduced a concept of a 'proposal age', and included a section in the spec doc that describe how we count blocks, and how age for proposal is calcuated. As a related change, I have also renamed `is_level_reached` function as `is_proposal_age`, to make it clear that it is checking the proposal age.

So in the comments is specification and types.ligo associated with config type, I have used this proposal age to describe the behavior. I think this might help to reduce ambiguity.